### DISCUSSION TITLE
DateTime.ToString() inconsistency between Windows and macOS

### DISCUSSION THREAD

**Discussion by danipen** [dotnet/runtime#97083] [General]
I have a simple code snippet in C#:

```csharp
Console.WriteLine("Hello, World!: " + DateTime.Now.ToString());
```

When I change the system settings, the output differs between Windows and macOS platforms.

On Windows, it respects the system settings as expected:
![image](https://github.com/dotnet/runtime/assets/501613/dbd05a85-68ee-4091-be10-47aeb30e6711)

However, on macOS, it doesn't follow the system settings:
![image](https://github.com/dotnet/runtime/assets/501613/106d064c-5616-4e52-9f60-dbe024755e2c)

Any insights or solutions would be appreciated. Thanks!

**Comment by huoyaoyuan** [comment]
See #61148

**Comment by tarekgh** [comment]
CC @steveisok
